common/scummsys.h

```diff
@@ -262,6 +262,14 @@
 //
 #if !defined(HAVE_CONFIG_H)
 
+	// If -fsanitize=undefined or -fsanitize=alignment is in use, and the
+	// compiler happens to report it, make sure SCUMM_NEED_ALIGNMENT is
+	// defined, in order to avoid false positives when not using the
+	// "configure" script to enable UBSan.
+	#if __has_feature(undefined_behavior_sanitizer)
```

Comment:
This is a clangism and would fail on GCC. You should check compiler first.
I don't know if this check must be done on an outer check or not, though.

Reply:
No, check the very top of scummsys.h there's a stub for non-Clang compilers ;)

Reply:
Oh! Well done!